index.js

```diff
@@ -178,12 +232,20 @@ class PQueue {
 	}
 
 	get pending() {
-		return this._pendingCount;
+		return this._concurrentCount;
 	}
 
 	get isPaused() {
 		return this._isPaused;
 	}
+
+	close() {
```

Comment:
I called it close because when I was using it the application would not automatically end until the interval was cleared (which could be up to 2n) so I needed a function to "close" it. Changing it to `clearInterval()` and adding a throws would make the behavior better defined and more clear.

Reply:
Also yeah, it should throw if the queue is not empty.